src/content/docs/en/guides/integrations-guide/sitemap.mdx

```diff
@@ -82,35 +82,70 @@ export default defineConfig({
 
 ## Usage
 
-`@astrojs/sitemap` requires a deployment / site URL for generation. Add your site's URL under your `astro.config.*` using the `site` property. This must begin with `http:` or `https:`.
+`@astrojs/sitemap` needs to know your site’s deployed URL to generate a sitemap.
 
-```js title="astro.config.mjs" "'https://stargazers.club'"
+Add your site's URL as the [`site`](/en/reference/configuration-reference/#site) option in `astro.config.mjs`. This must begin with `http://` or `https://`.
+
+```js title="astro.config.mjs" {5}
 import { defineConfig } from 'astro/config';
 import sitemap from '@astrojs/sitemap';
 
 export default defineConfig({
-  // ...
   site: 'https://stargazers.club',
   integrations: [sitemap()],
+  // ...
 });
 ```
 
-Note that unlike other configuration options, `site` is set in the root `defineConfig` object, rather than inside the `sitemap()` call.
+With the sitemap integration configured, `sitemap-index.xml` and `sitemap-0.xml` files will be added to your output directory when building your site.
 
-Now, [build your site for production](/en/reference/cli-reference/#astro-build) via the `astro build` command. You will find both `sitemap-index.xml` and `sitemap-0.xml` in the `dist/` folder (or your custom [output directory](/en/reference/configuration-reference/#outdir) if set).
+`sitemap-index.xml` links to all the numbered sitemap files.
+`sitemap-0.xml` lists the pages on your site.
+For extremely large sites, there may also be additional numbered files like `sitemap-1.xml` and `sitemap-2.xml`.
 
-:::caution
-If you forget to add a `site`, you'll get a friendly warning when you build, and the `sitemap-index.xml` file won't be generated.
-:::
+<details>
+<summary>Example of generated files for a two-page website</summary>
+
+```xml title="sitemap-index.xml"
+<?xml version="1.0" encoding="UTF-8"?>
+  <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <sitemap>
+    <loc>https://stargazers.club/sitemap-0.xml</loc>
+  </sitemap>
+</sitemapindex>
+```
+
+```xml title="sitemap-0.xml"
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
+  <url>
+    <loc>https://stargazers.club/</loc>
+  </url>
+  <url>
+    <loc>https://stargazers.club/second-page/</loc>
+  </url>
+</urlset>
+```
+</details>
+
+### Sitemap discovery
+
+You can make it easier for crawlers to find your sitemap with links in your site's `<head>` and `robots.txt` file.
```

Comment:
```suggestion
You can make it easier for crawlers to find your sitemap with links in your site's `<head>` or `robots.txt` file.
```
Just checking, would someone do *both* or only one of the following methods? If they're not supposed to add to both places (a conflict?) then I'd not use "and" this way. Could be swapping to "or" or could be extra context like:

> You can make it easier for crawlers to find your sitemap by including a link in either...

> To help crawlers discover your sitemap, you can add a link in one of the following locations...

Reply:
Yes, doing both is not uncommon. Do you still think “or” is better in that case?

Reply:
Ah, nope! If doing both is common, then it's fine as is!